course/src/main/java/in/testpress/course/fragments/ExamStartScreenFragment.kt

```diff
@@ -92,6 +98,7 @@ class ExamStartScreenFragment : BaseExamWidgetFragment() {
         showDescription()
         showOrHideExamDate(exam)
         showExamDuration(exam)
+        showExamStatusViews(exam)
```

Comment:
```suggestion
        showExamStatus(exam)
```

View keyword is obvious, so it is not necessary.

Reply:
Okay @Karthik-0, Will change this.